src/core/lua/luacfb.cpp

```diff
@@ -19,15 +19,14 @@
 #include "criticalregion.h"
 
 CLuaCFB::CLuaCFB(CStringDictionary::TStringId paInstanceNameId, const CLuaCFBTypeEntry* paTypeEntry, SCFB_FBNData &paFbnData, forte::core::CFBContainer &paContainer) :
-    CGenFunctionBlock<CCompositeFB>(paContainer, paTypeEntry->getInterfaceSpec(), paInstanceNameId, paFbnData), mTypeEntry(paTypeEntry) {
+    CGenFunctionBlock<CCompositeFB>(paContainer, paInstanceNameId, paFbnData), mTypeEntry(paTypeEntry) {
```

Comment:
I would suggest to change the return type of `CLuaCFBTypeEntry::getInterfaceSpec()` to a reference, keep passing it to the super constructor here and remove the assignment in `CLuaCFB::createInterfaceSpec()` instead.

Reply:
done